### PR TITLE
fixed tests: relying on ir_model_data record update was insufficent to…

### PR DESCRIPTION
**Pull request by rvalyi**
… set test data consistently after module install

### CODE REVIEW

**Comment by pedrobaeza** [comment]
:+1:

File: account_fiscal_position_rule/test/test_rules.yml [hunk 1]
```diff
@@ -65,6 +55,7 @@
       'partner_id': ref('base.res_partner_11'),
       'partner_invoice_id': ref('base.res_partner_11')
     }
+    self.env['res.partner'].browse(ref('base.main_company')).write({'country_id': ref('base.us')})
```
Comment (renatonlima) [inline account_fiscal_position_rule/test/test_rules.yml]: Do browse res.partner with ref('base.main_company')? the correct reference should be ref('base.main_partner')?

**Comment by rvalyi** [comment]
@renatonlima good catch, you are right! It worked by luck because both ids are 1 by default. I'm fixing this.

**Comment by renatonlima** [comment]
:+1:  LGTM

**Pull request merged**